modules/assisted-installer-how-to-use-the-assisted-installer.adoc

```diff
@@ -0,0 +1,32 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='how-to-use-the-assisted-installer_{context}']
+= How to use the Assisted Installer
+
+{product-title} {ai-full} is a user-friendly installation solution offered on link:http://console.redhat.com[console.redhat.com]. {ai-full} supports the various deployment platforms with a focus on bare metal. It comprises two key components:
```

Comment:
I'd link here the actual AI link you added at the bottom to access AI directly.

Also, when referring to cloud.redhat.com I'd call it as Red Hat Hybrid Cloud Console

Reply:
I've added both to the sentence.